scipy/stats/_continuous_distns.py

```diff
@@ -7040,11 +7040,31 @@ def _stats(self, df):
     def _entropy(self, df):
         if df == np.inf:
             return norm._entropy()
-        half = df/2
-        half1 = (df + 1)/2
-        return (half1*(sc.digamma(half1) - sc.digamma(half))
-                + np.log(np.sqrt(df)*sc.beta(half, 0.5)))
 
+        def regular(df):
+            half = df/2
+            half1 = (df + 1)/2
+            return (half1*(sc.digamma(half1) - sc.digamma(half))
+                    + np.log(np.sqrt(df)*sc.beta(half, 0.5)))
+
+        def asymptotic(df):
+            # gammaln(x) ~ x * ln(x) - x - 0.5 * ln(x) + 0.5 * ln(2 * pi)
+            # psi(x) ~ ln(x) - 1 / (2 * x)
+            # B(x, y) ~ gamma(x) * y ** (-x)
+            # (x / 2) * (psi(x + 1) - psi(x)) ~ 0.5
+            try:
+                E_1 = 1 / df + 1 / (4 * df ** 2) - 1 / (6 * df ** 3)
+                E_2 = -1 / (8 * df ** 4) + 3 / (10 * df ** 5) + 1 / (4 * df ** 6)
+
+            except RuntimeWarning:
+                E_1 = 0
+                E_2 = 0
+
+            h = norm._entropy() + E_1 + E_2
```

Comment:
```suggestion
            h = (norm._entropy() + 1./df + 1. / 4 * df**-2 - 1. / 6. * df**-3
              - 1. / 8. * df**-4 + 3. / 10. * df**-5 + 1 / 4 * df**-6)
```

Reply:
This should avoid overflow errors. When did the RuntimeWarning occur? Will hopefully also be avoided this way.

Reply:
The error occurred at 1e100, and would probably occur for all values larger than or equal to 1e60.